### DISCUSSION TITLE
How do I disable all other keyboard firmware and only leave the firmware I want in the workflow for zmk?

### DISCUSSION THREAD

**Discussion by glyh** [manna-harbour/miryoku#222] [Q&A] [answered]
As title, thanks!

**Comment by manna-harbour** [comment, marked as answer]
For help with build workflows see https://github.com/manna-harbour/miryoku_zmk#workflow-builds and in particular:
- https://github.com/manna-harbour/miryoku_zmk#build-examples
- https://github.com/manna-harbour/miryoku_zmk#build-inputs
- https://github.com/manna-harbour/miryoku_zmk#board
- https://github.com/manna-harbour/miryoku_zmk#shield

If that doesn't cover your issue please be more specific and provide examples.
